tools/filegone.py

```diff
@@ -9,6 +9,8 @@
 # Licensed under the Apache License, Version 2.0 (the "License")
 #
 # 08-Nov-2022 Curu. modified from filelife
+# 19-Nov-2022 Rong Tao vfs_rename/unlink/rmdir: Use CO-RE way, instead of
+#                      KERNEL_VERSION macro
```

Comment:
Just say 'Check btf struct field instead of KERNEL_VERSION macro'.
The subject can be 'Check btf struct field field of KERNEL_VERSION macro'.

The btf struct field checking applies to all three functions, not just vfs_rename().

Reply:
I changed the comment and subject, thanks Song.